gradio/chat_interface.py

```diff
@@ -81,7 +81,7 @@ def __init__(
         run_examples_on_click: bool = True,
         cache_examples: bool | None = None,
         cache_mode: Literal["eager", "lazy"] | None = None,
-        title: str | None = None,
+        title: str = "Gradio",
```

Comment:
We shouldn't change this actually b/c the `title` is also printed as Markdown on top of the Chat UI

Reply:
ahh, correct, thanks!